docs/tutorials/gromacs/spack-gromacs.md

```diff
@@ -0,0 +1,287 @@
+
+## Select a Project
+
+Select a project in which to deploy an HPC cluster on Google.
+
+<walkthrough-project-setup billing="true"></walkthrough-project-setup>
+
+Once you have selected a project, click START.
+
+## Enable APIs & Permissions
+
+_Skip this step if you already ran this as part of a previous tutorial._
+
+In a new Google Cloud project there are several apis that must be enabled to
+deploy your HPC cluster. These will be caught when you perform `terraform apply`
+but you can save time by enabling them now by running:
+
+<!-- Tried the native way to do this and it timed out. Leaving comment here for future reference. -->
+<!-- <walkthrough-enable-apis apis="file.googleapis.com,compute.googleapis.com"></walkthrough-enable-apis> -->
+
+```bash
+gcloud services enable --project <walkthrough-project-id/> file.googleapis.com compute.googleapis.com
+```
+
+We also need to grant the default compute service account project edit access so
+the slurm controller can perform actions such as auto-scaling.
+
+<!-- Tried getting PROJECT_NUMBER using <walkthrough-project-number/> but returns empty string. -->
+
+```bash
+PROJECT_NUMBER=$(gcloud projects list --filter=<walkthrough-project-id/> --format='value(PROJECT_NUMBER)')
+
+echo "granting roles/editor to $PROJECT_NUMBER-compute@developer.gserviceaccount.com"
+
+gcloud iam service-accounts enable --project <walkthrough-project-id/> "$PROJECT_NUMBER"-compute@developer.gserviceaccount.com
+
+gcloud projects add-iam-policy-binding <walkthrough-project-id/> --member=serviceAccount:"$PROJECT_NUMBER"-compute@developer.gserviceaccount.com --role=roles/editor
+```
+
+## Build the Toolkit Binary
+
+*Skip this step if you already ran this as part of a previous tutorial.*
+
+To build HPC Toolkit binary from source run:
+
+```bash
+make
+```
+
+You should now have a binary named ghpc in the current directory. To verify the
+build run:
+
+```bash
+./ghpc --version
+```
+
+This should show you the version of the HPC Toolkit you are using.
+
+## Generate a Deployment
+
+This tutorial will use the blueprint docs/tutorials/gromacs/spack-gromacs.yaml,
+which should be open in the Cloud Shell Editor (on the left).
+
+This file describes the cluster you will deploy. It defines:
+
+* the existing default network from your project
+* a definition of a custom Spack installation
+* a startup script that
+  * installs ansible
+  * installs Spack & Gromacs using the definition above
+  * sets up a Spack environment including downloading an example input deck
+  * places a submission script on a shared drive
+* a Slurm cluster
+  * a Slurm login node
+  * a Slurm controller
+  * An auto-scaling Slurm partition
+
+After you have inspected the file, use the ghpc binary to create a deployment
+folder by running:
+
+```bash
+./ghpc create community/examples/workshops/spack-gromacs.yaml --vars project_id=<walkthrough-project-id/> --vars spack_cache_mirror_url=https://storage.googleapis.com/pnnl-clouddays22-workshop
```

Comment:
We need to make sure we remove the buildcache URL before publishing this, and then we add it back in.

Reply:
I can think of a few ways to execute on this, let's discuss.

Reply:
Done.